xcode/common.xcconfig

```diff
@@ -4,8 +4,8 @@ QT_PATH[arch=x86_64] = /usr/local/opt/qt@5
 QT_PATH[arch=arm64] = /opt/homebrew/opt/qt@5
 BOOST_ROOT = $(OMIM_ROOT)/3party/boost
 
-// jansson is included in many libs, and is also used in headers (leaks to other libs)
-HEADER_SEARCH_PATHS = $(inherited) $(OMIM_ROOT) $(BOOST_ROOT) $(OMIM_ROOT)/3party/jansson $(OMIM_ROOT)/3party/jansson/jansson/src
+// jansson and protobuf are included in many libs, and are also used in headers (leaks to other libs)
+HEADER_SEARCH_PATHS = $(inherited) $(OMIM_ROOT) $(BOOST_ROOT) $(OMIM_ROOT)/3party/jansson $(OMIM_ROOT)/3party/jansson/jansson/src $(OMIM_ROOT)/3party/protobuf/protobuf/third_party/utf8_range/ $(OMIM_ROOT)/3party/protobuf/protobuf/third_party/abseil-cpp/ $(OMIM_ROOT)/3party/protobuf/protobuf/src
```

Comment:
Are abseil-cpp includes required in our case? What features do they provide for the protobuf?

Reply:
Probably, we can avoid putting protobuf and dependent stuff here.
Enough to add them to indexer, drape_frontend, map, some tests ..
Where drules_include.hpp is included.